test/integration/src/main/java/io/pravega/test/integration/selftest/Event.java

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.test.integration.selftest;
+
+import com.google.common.base.Preconditions;
+import io.pravega.common.io.StreamHelpers;
+import io.pravega.common.util.ArrayView;
+import io.pravega.common.util.BitConverter;
+import io.pravega.common.util.ByteArraySegment;
+import io.pravega.test.integration.selftest.adapters.TruncateableArray;
+import java.io.IOException;
+import lombok.Getter;
+import lombok.SneakyThrows;
+
+/**
+ * Represents an Event with a Routing Key and payload.
+ */
+public class Event {
```

Comment:
This is a new class. I moved some code over from another class, but I couldn't have renamed the other class since it was doing so many other things.